### PR TITLE
docs: fixes to versioning strategy

### PR DESCRIPTION
**Pull request by antusus**

### CODE REVIEW

File: content/guides/api-calls/api-versioning-strategy.md [hunk 1]
```diff
@@ -87,6 +87,32 @@ The client gets a list of all created sign requests and asks for version `2025.0
...
+  },
+  "help_url": "https://developer.box.com/guides/api-calls/permissions-and-errors/versioning-errors/"
+}
+```
+
+The error message will contain information about the error and possible correct values for the `box-version` header. For example:
+
+ - `The 'box-version' header supports only one header value per request, do not use comas.` - when the header contains multiple values separated by commas.
+ - `Missing required box-version header.` - when the header is missing but required.
+ - `Unsupported API version specified in 'box-version' header. Supported API versions: [LIST_OF_SUPPORTED_VERSIONS_COMA_SEPARATED]` - when the version specified is not supported by the API.
```
Comment (wsocha) [inline content/guides/api-calls/api-versioning-strategy.md:114]: [This paragraph](https://github.com/box/developer.box.com/blob/main/content/guides/api-calls/api-versioning-strategy.md#calling-a-nonexistent-version) should be removed as it falls into this category

Comment (wsocha) [reply content/guides/api-calls/api-versioning-strategy.md:114]: The same applies to [this one](https://github.com/box/developer.box.com/blob/main/content/guides/api-calls/api-versioning-strategy.md#calling-an-incorrect-api-version-in-the-url).

Comment (wsocha) [reply content/guides/api-calls/api-versioning-strategy.md:114]: Can you review what from `Versioning errors` in the current form should be left?

Comment (wsocha) [reply content/guides/api-calls/api-versioning-strategy.md:114]: This paragraph seems also outdated: https://github.com/box/developer.box.com/blob/main/content/guides/api-calls/api-versioning-strategy.md?plain=1#L218

**Review by Percival33 — APPROVED** [review]

**Review by wsocha — APPROVED** [review]

**Pull request merged**